opencti-platform/opencti-front/src/utils/defaultRepresentatives.ts

```diff
@@ -96,42 +96,42 @@ export const getMainRepresentative = (n: any, fallback = 'Unknown') => {
     return defaultValueMarking(n);
   }
   const mainValue: string = n.representative?.main
-    || n.name
-    || n.label
-    || n.observableName
-    || n.observable_value
-    || n.pattern
-    || n.attribute_abstract
-    || n.opinion
-    || n.value
-    || n.definition
-    || n.source_name
-    || n.phase_name
-    || n.result_name
-    || n.country
-    || n.key
-    || (n.template && n.template.name)
-    || (n.content && truncate(n.content, 30))
-    || (n.hashes
-      && (n.hashes.MD5
-        || n.hashes['SHA-1']
-        || n.hashes['SHA-256']
-        || n.hashes['SHA-512']))
-    || (n.source_ref_name
-      && n.target_ref_name
-      && `${truncate(n.source_ref_name, 20)} ➡️ ${truncate(
-        n.target_ref_name,
-        20,
-      )}`)
-    || getMainRepresentative((R.head(R.pathOr([], ['objects', 'edges'], n)) as any)?.node)
-    || (n.from
-      && n.to
-      && `${truncate(getMainRepresentative(n.from), 20)} ➡️ ${truncate(
-        getMainRepresentative(n.to),
-        20,
-      )}`)
-    || n.main_entity_name
-    || fallback;
+      || n.name
```

Comment:
did something change in this file? I only see reformatting

Reply:
And I think it is not the right formatting as we are using a 2 spaces formats in other files, not 4